content/php/concepts/string-functions/terms/rtrim/rtrim.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'rtrim()'
+Description: 'removes whitespace or other predefined characters from the right side of a string.'
+Subjects: 
+    - 'Computer Science'
+    - 'Web Design'
+    - 'Web Development'
+Tags: 
+    - 'Strings'
+    - 'Functions'
+CatelogContent:
+    - 'learn-php'
+    - 'paths/computer-science'
+---
+
+The **`rtrim()`** method removes whitespace or other predefined characters from the right side of a string.
+
+## Syntax
+```psuedo
+rtrim(string,charlist)
+```
+
+- The `string` parameter is the string from which we need to remove the characters.
+- The `charlist` parameter specifies which characters to remove from the string. If omitted then following characters are removed.
+
+ - **"\O"** - NULL
+ - **"\t"** - tab
+ - **"\n"** - new line
+ - **"xOB"** - vertical tab
+ - **"\r"** - carriage return
+ - **" "** - whitespace
+
+## Codebyte Example 
```

Comment:
We generally only have one codebyte example per entry. Remove one of these, or make one a regular `## Example` with sample output.

Reply:
made one a regular ## Example with sample output